### ISSUE TITLE
Bug report - long bars with 0 

### ISSUE THREAD

**Issue by kfagnan** [microbiomedata/nmdc-server#268]
I think the shaded areas represent the total, but with 0 next to the end, it looks a bit strange. Maybe representing as a fraction 0/X would help show what is meant?

![image (1)](https://user-images.githubusercontent.com/1504015/107835904-fe3a2700-6d4f-11eb-8f27-a614695e5105.png)

Feedback:
Bug report.

Why the first "0" comes with such a long bar? Are the numbers correct?

**Comment by jeffbaumes** [comment]
You are correct in the interpretation. Yes we could do 0 / X for clarity.

As for the long bar, there are many times more organic matter omics_processing than everything else, due to the fact that it is run on several subsamples (10-15 per sample) and each counts as an omics_processing.

**Comment by subdavis** [comment]
![Screenshot from 2021-02-18 11-07-55](https://user-images.githubusercontent.com/4214172/108412303-b441bd00-71f7-11eb-9f49-c17e08e283b9.png)
![Screenshot from 2021-02-18 11-07-00](https://user-images.githubusercontent.com/4214172/108412304-b4da5380-71f7-11eb-95ad-3b95ae777e02.png)


I tried both, and I found adding the totals to be "noisy".  It changed my ability to glance at the number to having to focus and read.  I tried "No match" instead.  Let me know what you think.
